io/net/port_bounce.py.data/README.txt

```diff
@@ -0,0 +1,13 @@
+Port bounce Testcase:
+
+this testcase mainly depends on the setup of fc or fcoe switch to
+connect different adapters to different ports in respective switches.
+
```

Comment:
@abdhaleegit 2 things to note here:

1. The current implementation will take care of this, if given storage port ids instead of host port ids.
2.  The problem with doing that is, all the other FC hosts will be affected by disabling/enabling storage ports. So, that should be handled with caution.